SPECS/go-md2man/go-md2man.spec

```diff
@@ -42,24 +40,29 @@ mkdir -p "%{buildroot}%{_bindir}"
 cp -aT go-md2man-2.0.0/go-md2man %{buildroot}%{_bindir}/go-md2man
 
 # copy legal files
-mkdir -p %{buildroot}/usr/share/doc/%{name}-%{version}
-cp go-md2man-2.0.0/LICENSE.md %{buildroot}/usr/share/doc/%{name}-%{version}/LICENSE.md
+mkdir -p %{buildroot}%{_docdir}/%{name}-%{version}
+cp go-md2man-2.0.0/LICENSE.md %{buildroot}%{_docdir}/%{name}-%{version}/LICENSE.md
 
 %files
-%license /usr/share/doc/%{name}-%{version}/LICENSE.md
+%license %{_docdir}/%{name}-%{version}/LICENSE.md
```

Comment:
Looks OK, but would a simple `LICENSE.md` without the whole path work?

Reply:
It won't work without the whole path.